gazebo/physics/Actor_TEST.cc

```diff
@@ -102,6 +102,62 @@ TEST_F(ActorTest, TrajectoryFromSDF)
   EXPECT_LT(fabs(actor->ScriptTime() - world->SimTime().Double()), 1.0 / 30);
 }
 
+//////////////////////////////////////////////////
+TEST_F(ActorTest, ActorCollision)
+{
+  // Load a world with an actor
+  this->Load("examples/plugins/actor_collisions/actor_collisions_noloop.world",
+             true);
+  auto world = physics::get_world("default");
+  ASSERT_TRUE(world != nullptr);
+
+  // Get model
+  auto model = world->ModelByName("actor");
+  ASSERT_TRUE(model != nullptr);
+
+  // Convert to actor
+  auto actor = boost::dynamic_pointer_cast<physics::Actor>(model);
+  ASSERT_TRUE(actor != nullptr);
+
+  // Check collisions are enabled for joints defined in .world
+  std::vector<std::string> linkNames = {"LHipJoint", "LeftUpLeg", "LeftLeg",
+                                        "LeftFoot", "RHipJoint", "RightUpLeg",
+                                         "RightLeg", "RightFoot", "LowerBack",
+                                         "Spine", "Neck", "Neck1",
+                                         "LeftShoulder", "RightShoulder",
+                                         "LeftArm", "LeftForeArm",
+                                         "RightArm", "RightForeArm",
+                                         "LeftFingerBase", "RightFingerBase"};
+  for (const auto &name : linkNames)
+  {
+    auto link = actor->GetLink(name);
```

Comment:
Just in case something changes in the actor model, it would be good to `ASSERT_NE(nullptr, link);` before using the link pointer below. Otherwise the test will crash.